2025-04-03T21:55:31.624+0000 [INFO] Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: iis_etl.archive scheduled__2025-04-03T21:54:00.504138+00:00 [queued]>
2025-04-03T21:55:31.634+0000 [INFO] Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: iis_etl.archive scheduled__2025-04-03T21:54:00.504138+00:00 [queued]>
2025-04-03T21:55:31.634+0000 [INFO] Starting attempt 1 of 4
2025-04-03T21:55:31.643+0000 [INFO] Executing <Task(PythonOperator): archive> on 2025-04-03 21:54:00.504138+00:00
2025-04-03T21:55:31.648+0000 [INFO] Started process 1252 to run task
2025-04-03T21:55:31.651+0000 [INFO] Running: ['airflow', 'tasks', 'run', 'iis_etl', 'archive', 'scheduled__2025-04-03T21:54:00.504138+00:00', '--job-id', '404', '--raw', '--subdir', 'DAGS_FOLDER/iis_etl.py', '--cfg-path', '/tmp/tmpo6hflx34']
2025-04-03T21:55:31.653+0000 [INFO] Job 404: Subtask archive
2025-04-03T21:55:31.726+0000 [INFO] Running <TaskInstance: iis_etl.archive scheduled__2025-04-03T21:54:00.504138+00:00 [running]> on host 8d34dc648251
2025-04-03T21:55:32.355+0000 [INFO] Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='iis_etl' AIRFLOW_CTX_TASK_ID='archive' AIRFLOW_CTX_EXECUTION_DATE='2025-04-03T21:54:00.504138+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-03T21:54:00.504138+00:00'
2025-04-03T21:55:32.363+0000 [WARNING] File /opt/airflow/data/processing/u_ex100718.log does not exist for archiving
2025-04-03T21:55:32.364+0000 [INFO] Done. Returned value was: None
2025-04-03T21:55:32.372+0000 [INFO] Marking task as SUCCESS. dag_id=iis_etl, task_id=archive, execution_date=20250403T215400, start_date=20250403T215531, end_date=20250403T215532
2025-04-03T21:55:32.411+0000 [INFO] Task exited with return code 0
2025-04-03T21:55:32.444+0000 [INFO] 0 downstream tasks scheduled from follow-on schedule check
